### PR TITLE
Adds Itch.io Butler

### PR DESCRIPTION
**Pull request by xuv**
Still WIP and untested the deployement so far.
But I've successfully build the Docker image locally.
I thought I'd submit for PR so you can have a look. 
This is my first docker container, so not sure I got all the things right.
Also there will probably need some more explanations in the documentation to set that up properly.
Let me know what you think.

### CODE REVIEW

**Comment by aBARICHELLO** [comment]
Hey there, thanks for the PR. 
You can edit it to a draft PR and back to a normal one when it's done if you want. This way I can be properly notified.

(Closes #5)

**Comment by xuv** [comment]
Thanks. I can't seem to find the option to edit it back to a draft PR. Unless, I guess I recreate the PR from scratch. Do you prefer I do this?

**Comment by aBARICHELLO** [comment]
It's OK, just tag me/add as a reviewer when ready.

**Comment by xuv** [comment]
Ok, I think it's ready. 
I've been using this in this repo: https://gitlab.com/polymorphcool/ormer (the deployment have worked successfully on the `ci` branch there)

Let me know what you think

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ stages:
 
 variables:
   EXPORT_NAME: game-off
+  ITCHIO_USERNAME: user
+  ITCHIO_GAME: game
```
Comment (aBARICHELLO) [inline .gitlab-ci.yml]: Since itch.io is optional i think these variables should be included in the gitlab ci panel

Comment (xuv) [reply .gitlab-ci.yml]: Good idea. I fixed it in the last commit and updated the documentation

**Comment by aBARICHELLO** [comment]
Just tested your PR and it worked OK. Thanks for the contribution!

**Pull request merged**